featherbed-core/src/main/scala/featherbed/request/RequestSyntax.scala

```diff
@@ -68,6 +70,77 @@ trait RequestTypes { self: Client =>
       canBuild: CanBuildRequest[Self]
     ): ValidatedNel[Throwable, Request] = canBuild.build(this: Self)
 
+    private def cloneRequest(in: Request) = {
+      val out = Request()
+      out.uri = in.uri
+      out.content = in.content
+      in.headerMap.foreach {
```

Comment:
Cookies? Copying HTTP requests in Finagle is quite a pain (something things we can miss out). Any way we can avoid this?